### PR TITLE
Add unit tests for hello module

### LINKED ISSUE #1: Add input validation to hello function
The hello function should validate that the name parameter is not empty.

## Steps to reproduce
1. Call hello with empty string
2. Returns empty greeting

### PR DESCRIPTION
**Pull request by richard-pokee**
This PR adds unit tests for the hello module.

- test_hello_default
- test_hello_name

---

Closes #1

### CODE REVIEW

**Review by richard-pokee — COMMENTED** [review]
Looks good! Tests cover both scenarios.

File: tests/test_hello.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"""Tests for hello module."""
+from hello import hello
+
+
+def test_hello_default():
+    assert hello() == "Hello, World!"
```
Comment (richard-pokee) [inline tests/test_hello.py:6]: Consider adding a test for edge case with empty string input.

File: tests/test_hello.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+
+def test_hello_name():
```
Comment (richard-pokee) [inline tests/test_hello.py:9]: Use a more descriptive test name

```suggestion
    assert hello("Pokee Bot") == "Hello, Pokee Bot!"
```
